doc/tutorials/serve.md

```diff
@@ -0,0 +1,329 @@
+# Serving Panel Apps
+
+In this section you will learn how to serve Panel apps
+
+## Serve an app from a `.py` file
+
+The simplest Panel `.py` file could look like this:
+
+```python
+import panel as pn
```

Comment:
How do I avoid having the output of `pn.panel("Hello World").servable() displayed in the docs?

Reply:
append `;`?

Reply:
Sorry. That was not what I meant. But I learned that ```python is not rendered while ```{pyodide}``` is.